src/util/validationUtils.js

```diff
@@ -78,12 +78,22 @@ const validateCareGapsParams = query => {
       );
     }
   } else if (query.subject) {
+    if (query.practitioner) {
+      throw new BadRequestError('Must provide either subject or practitioner. Received both');
+    }
     const subjectReference = query.subject.split('/');
     if (subjectReference.length !== 2 || !['Group', 'Patient'].includes(subjectReference[0])) {
       throw new BadRequestError(
         `Subject may only be a Group resource of format "Group/{id}" or Patient resource of format "Patient/{id}".`
       );
     }
+  } else if (query.practitioner) {
```

Comment:
This case should actually throw an error, since we can only reach here if a practitioner was provided without and organization. I would move the contents of this conditional inside the conditional that checks for an organization being passed, and just have this conditional throw an error saying practitioner must be passed in with organization